ee/ucp/interlock/usage/service-clusters.md

```diff
@@ -4,205 +4,257 @@ description: Learn how to route traffic to different proxies using a service clu
 keywords: ucp, interlock, load balancing, routing
 ---
 
-## Configure Proxy Services
-With the node labels, you can re-configure the Interlock Proxy services to be constrained to the
-workers for each region. For example, from a manager, run the following commands to pin the proxy services to the ingress workers:
+Reconfiguring Interlock's proxy can take a long time (1-2 seconds) per overlay network managed by that proxy. In order to scale up to larger number of Docker networks and services routed to by Interlock, you may consider implementing *service clusters*: multiple proxy services managed by Interlock (rather than the default single proxy service), each responsible for routing to a separate set of Docker services and their corresponding networks, thereby minimizing proxy reconfiguration time.
 
-```bash
-$> docker service update \
-    --constraint-add node.labels.nodetype==loadbalancer \
-    --constraint-add node.labels.region==us-east \
-    ucp-interlock-proxy-us-east
-$> docker service update \
-    --constraint-add node.labels.nodetype==loadbalancer \
-    --constraint-add node.labels.region==us-west \
-    ucp-interlock-proxy-us-west
-```
-
-You are now ready to deploy applications. First, create individual networks for each application:
+## Prerequisites
 
-```bash
-$> docker network create -d overlay demo-east
-$> docker network create -d overlay demo-west
-```
+In this example, we'll assume you have a UCP cluster set up with at least two worker nodes, `ucp-node-0` and `ucp-node-1`; we'll use these as dedicated proxy servers for two independent Interlock service clusters. 
 
-Add the networks to the Interlock configuration file. Interlock automatically adds networks to the proxy service upon the next proxy update. See *Minimizing the number of overlay networks* in [Interlock architecture](https://docs.docker.com/ee/ucp/interlock/architecture/) for more information. 
+We'll also assume you've already enabled Interlock, per the [instructions here](https://docs.docker.com/ee/ucp/interlock/deploy/), with an HTTP port of 80 and an HTTPS port of 8443.
 
-> Note
-> 
-> Interlock will _only_ connect to the specified networks, and will connect to them all at startup.
+## Setting Up Interlock Service Clusters
```

Comment:
Sentence case for titles.

##Setting up Interlock service clusters
(assuming Interlock is a proper noun)